pkg/statistics/table.go

```diff
@@ -819,16 +829,18 @@ func (t *Table) ColumnIsLoadNeeded(id int64, fullLoad bool) (*Column, bool, bool
 	if t.Pseudo {
 		return nil, false, false
 	}
-	// when we use non-lite init stats, it cannot init the stats for common columns.
-	// so we need to force to load the stats.
 	col, ok := t.columns[id]
+	hasAnalyzed := t.ColAndIdxExistenceMap.HasAnalyzed(id, false)
```

Comment:
```suggestion
	hasAnalyzed := t.ColAndIdxExistenceMap.HasAnalyzed(id, false)
	col, ok := t.columns[id]
```

Placing the `ok` closer to the `ok` check would make the code easier to read.